x-pack/plugins/observability/public/components/app/observability_status/observability_status_box.tsx

```diff
@@ -81,50 +94,62 @@ export function CompletedStatusBox({
           </EuiFlexItem>
         ))}
       </EuiFlexGroup>
+
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiButton color="primary" href={goToAppLink}>
+            {goToAppTitle}
+          </EuiButton>
+        </EuiFlexItem>
+      </EuiFlexGroup>
     </EuiPanel>
   );
 }
 
 export function EmptyStatusBox({
-  dataSourceName,
+  title,
   description,
   learnMoreLink,
+  addTitle,
+  addLink,
 }: ObservabilityStatusBoxProps) {
   return (
-    <EuiPanel color="subdued" style={{ marginBottom: 20 }}>
+    <EuiPanel color="warning" hasBorder={true}>
       <EuiFlexGroup justifyContent="spaceBetween">
         <EuiFlexItem>
           <div>
             <EuiIcon
-              type="dot"
+              type="minusInCircleFilled"
               color="warning"
               className="eui-displayInline eui-alignMiddle"
               style={{ marginRight: 8 }}
             />
             <EuiTitle size="xs" className="eui-displayInline eui-alignMiddle">
-              <h2>{dataSourceName}</h2>
+              <h2>{title}</h2>
             </EuiTitle>
           </div>
         </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiIcon type="cross" color="text" />
-        </EuiFlexItem>
       </EuiFlexGroup>
 
       <EuiFlexGroup>
         <EuiFlexItem>
-          <EuiText size="xs">{description}</EuiText>
+          <EuiText size="s">{description}</EuiText>
         </EuiFlexItem>
       </EuiFlexGroup>
 
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiButton color="primary" size="s" href={learnMoreLink}>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem grow={false}>
+          <EuiButton color="primary" href={addLink} target="_blank" fill>
```

Comment:
```suggestion
          <EuiButton color="primary" href={addLink} fill>
```
I don't think our internal links should be in a new window e.g. for add integrations or add data for APM. It will load in Kibana anyhow.